CONTRIBUTING.md

```diff
@@ -49,6 +49,7 @@ plumbing: packp, Skip argument validations for unknown capabilities. Fixes #623
 ```bash
 $ go get -d -u github.com/src-d/code-annotation/...
 $ cd $GOPATH/github.com/src-d/code-annotation
+$ touch .env
```

Comment:
Now make serve will fail because the OAUTH_CLIENT_ID is missing.

Moreover, the app won't do much without github auth, so I think setting up a real auth is a requirement also for development. We should either copy, or link to, the 'Github OAuth tokens' section of the README

Reply:
my initial intention was

1) You read readme and run project
2) Then you can use this commands to run it in dev mode.

Do you think something like "Make sure you are able to run the project by following steps from README" at the start of Development section will be enough?

Reply:
Yes. What about this?

Please note: you will need a `.env` file configured with working GitHub OAuth credentials to run the application in development mode. Please follow the [README Installation section](./README.md#installation) for instructions on how to do it.

Reply:
fixed